packages/vat-data/src/exo-utils.js

```diff
@@ -6,17 +6,17 @@ import { provide, VatData as globalVatData } from './vat-data-bindings.js';
 
 /** @template L,R @typedef {import('@endo/eventual-send').RemotableBrand<L, R>} RemotableBrand */
 /** @template T @typedef {import('@endo/far').ERef<T>} ERef */
-/** @typedef {import('./types.js').Baggage} Baggage */
-/** @template T @typedef {import('./types.js').DefineKindOptions<T>} DefineKindOptions */
-/** @template T @typedef {import('./types.js').KindFacet<T>} KindFacet */
-/** @template T @typedef {import('./types.js').KindFacets<T>} KindFacets */
-/** @typedef {import('./types.js').DurableKindHandle} DurableKindHandle */
+/** @typedef {import('@agoric/swingset-liveslots/src/vatDataTypes').Baggage} Baggage */
+/** @template T @typedef {import('@agoric/swingset-liveslots/src/vatDataTypes').DefineKindOptions<T>} DefineKindOptions */
+/** @template T @typedef {import('@agoric/swingset-liveslots/src/vatDataTypes').KindFacet<T>} KindFacet */
+/** @template T @typedef {import('@agoric/swingset-liveslots/src/vatDataTypes').KindFacets<T>} KindFacets */
+/** @typedef {import('@agoric/swingset-liveslots/src/vatDataTypes').DurableKindHandle} DurableKindHandle */
```

Comment:
I suspect I know the answer to this, but is there any way to code this type info that isn't so verbose and ugly?

Reply:
Yes! a little: https://github.com/Agoric/agoric-sdk/pull/8139/commits/3103f0a46a487ac13b02f2dbc70f60bb9214c5db

To avoid `import` on each lined we'd need to be able to import to module scope. We can't do that without `.ts` syntax. (Another option is duplicating the type locally, but we're not doing that anymore.)